ctru-rs/src/services/fs.rs

```diff
@@ -6,34 +6,23 @@
 #![doc(alias = "filesystem")]
 
 use bitflags::bitflags;
-use std::ffi::OsString;
-use std::io::Error as IoError;
-use std::io::ErrorKind as IoErrorKind;
-use std::io::Result as IoResult;
-use std::io::{Read, Seek, SeekFrom, Write};
-use std::mem;
-use std::path::{Path, PathBuf};
-use std::ptr;
-use std::slice;
-use std::sync::Arc;
-use widestring::{WideCStr, WideCString};
 
 bitflags! {
     #[derive(Default, PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Clone, Copy)]
-    struct FsOpen: u32 {
+    struct Open: u32 {
```

Comment:
Are all of the remaining enums/bitflags used somewhere?

Reply:
Not all, but it makes sense to keep them around just for some nice compatibility with `ctru_sys` (in case some end-user needs it).